go/test/endtoend/cluster/cluster_util.go

```diff
@@ -20,11 +20,15 @@ import (
 	"context"
 	"fmt"
 	"os"
+	"path"
+	"reflect"
 	"strings"
 	"testing"
 	"time"
 
+	"vitess.io/vitess/go/json2"
 	replicationdatapb "vitess.io/vitess/go/vt/proto/replicationdata"
+	"vitess.io/vitess/go/vt/proto/topodata"
```

Comment:
this is a dupe of the (_very_ weirdly) renamed `tabletpb` on line 34/38

do you want to tidy up these imports into the usual sections "while you're here"? (please feel free to say "no"!!)